common/flavorization/wc_title_holders.txt

```diff
@@ -4,12 +4,51 @@
 @county_cultural_prio = 18
```

Comment:
It's tribal.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/122430388-9c5f2780-cfa4-11eb-9972-73b62f07f9b3.png)

</details>

Reply:
I can't find why it do that but i fix with another way in 71fb75d0d

Reply:
> I can't find why it do that but i fix with another way in 71fb75d0d

Link not working.